pkg/rules/rulespb/rpc.proto

```diff
@@ -126,6 +127,7 @@ message Alert {
     string last_error                         = 9 [(gogoproto.jsontag) = "lastError,omitempty" ];
     double evaluation_duration_seconds        = 10 [(gogoproto.jsontag) = "evaluationTime" ];
     google.protobuf.Timestamp last_evaluation = 11 [(gogoproto.jsontag) = "lastEvaluation", (gogoproto.stdtime) = true, (gogoproto.nullable) = false ];
+    google.protobuf.Duration keep_firing_for = 12 [(gogoproto.jsontag) = "keepFiringFor", (gogoproto.stdduration) = true, (gogoproto.nullable) = false ];
```

Comment:
I think we need to add `keep_firing_since` to AlertInstance as well here. I now notice some fields are missing from Prometheus upstream too. Will raise a PR to fix.

Reply:
Will add it to AlertInstance as well.Regarding the type, since on prometheus side its duration i have given the same here as well.